### PR TITLE
change design workspaceList and createWorkspace

### PR DESCRIPTION
**Pull request by Sergey988**
1. Changed design the workspaceList page
2. Changed the "addWorkspace" page component to the "addWorkspace" modal window component

### CODE REVIEW

**Comment by Sergey988** [comment]
Some mistakes was fixed in the next commits. Sorry about this ))

File: src/app/core/models/userWorkspaceList.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export class UserWorkspace {
-    id?: number;
-    name?: string;
-    role?: string;
+    id: number;
+    name: string;
+    role: string;
 }
```
Comment (YevheniiVovk) [inline src/app/core/models/userWorkspaceList.ts]: Empty line at end of file

File: src/app/presentation/components/add-workspace/add-workspace.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<div class="container">
-  <div class="form-section">
+<div class="create-workspace">
+  <div class="">
```
Comment (YevheniiVovk) [inline src/app/presentation/components/add-workspace/add-workspace.component.html]: Why class is empty ?

File: src/app/presentation/components/workspace-list/workspace-list.component.html [hunk 3]
```diff
@@ -2,19 +2,30 @@
...
-      <mat-card *ngFor="let workspace of userWorkspaceList" [routerLink]="'../workspace/'+workspace.id">
-        <mat-card-title>
-          {{workspace.name}}
-        </mat-card-title>
-        <mat-card-subtitle >
-          Your role: {{workspace.role}}
-        </mat-card-subtitle>
-      </mat-card>
+      <div class="list">
+        <mat-card *ngFor="let workspace of userWorkspaceList" [routerLink]="'../workspace/'+workspace.id" class="card" [style.border-color]="colorArray[workspace.id % colorArray.length]">
```
Comment (YevheniiVovk) [inline src/app/presentation/components/workspace-list/workspace-list.component.html]: After + should be a whitespace

File: src/app/presentation/components/workspace-list/workspace-list.component.html [hunk 4]
```diff
@@ -2,19 +2,30 @@
...
+              <h3>{{workspace.name}}</h3>
+            </mat-card-title>
+            <mat-card-subtitle >
+              <span>Your role: {{workspace.role}}</span>
+            </mat-card-subtitle>
+          </div>
+          <div>
+            <mat-card-content class="footer-card-block" [style.color]="colorArray[workspace.id % colorArray.length]">
+              <span>Active Task: 11</span>
+              <span>23/12/2021</span>
```
Comment (YevheniiVovk) [inline src/app/presentation/components/workspace-list/workspace-list.component.html]: Why date is const value ?

File: src/app/presentation/components/workspace-list/workspace-list.component.ts [hunk 5]
```diff
@@ -1,19 +1,35 @@
...
   selector: 'app-workspace-list',
   templateUrl: './workspace-list.component.html',
   styleUrls: ['./workspace-list.component.css']
 })
 export class WorkspaceListComponent implements OnInit {
+  
   userWorkspaceList: UserWorkspace[];
-  constructor(private workspaceService: WorkspaceService) {}
-
+  colorArray: string[] = ['#666ec3','#8b3d3d','#4c9763','#3c3c3c','#974d0e','#529992'];
```
Comment (YevheniiVovk) [inline src/app/presentation/components/workspace-list/workspace-list.component.ts]: Let's move it out from here

File: src/app/presentation/components/workspace-list/workspace-list.component.html (hunk 4 above)
Comment (Sergey988) [reply src/app/presentation/components/workspace-list/workspace-list.component.html]: Backend functionality is not ready yet

**Review by YevheniiVovk — APPROVED** [review]

**Review by YevheniiVovk — APPROVED** [review]

**Review by vladtymo — APPROVED** [review]

**Pull request merged**